hpfeeds-cif/feedhandler.py

```diff
@@ -194,13 +196,13 @@ def main():
     processor = processors.HpfeedsMessageProcessor(ignore_cidr_list=ignore_cidr_l)
     logging.debug('Initializing HPFeeds connection with {0}, {1}, {2}, {3}'.format(host,port,ident,secret))
     try:
-        hpc = hpfeeds.new(host, port, ident, secret)
+        hpc = hpfeeds.client.new(host, port, ident, secret)
     except hpfeeds.FeedException as e:
         logging.error('Experienced FeedException: {0}'.format(repr(e)))
         return 1
 
     def on_message(identifier, channel, payload):
-        for msg in processor.process(identifier, channel, payload, ignore_errors=True):
+        for msg in processor.process(identifier, channel, payload.decode('utf-8'), ignore_errors=True):
```

Comment:
In other spots you've removed the utf-8 encode commands, but here you add a decode...does this mean that something in another space is performing an encode we need to change there?

Reply:
It's because the payload type coming out of hpfeeds3 is bytes instead of string, so you need to decode the payload to a string. This wasn't a problem with python2, since python2 doesn't force strict type checking on strings / bytes.